pkgs/development/interpreters/python/pypy/prebuilt_2_7.nix

```
@@ -48,24 +50,35 @@ let
 
   deps = [
     bzip2
+    gdbm
+    sqlite
     zlib
     expat
     ncurses6
     tcl-8_5
     tk-8_5
   ];
 
+  downloadUrls = {
+    aarch64-linux = "https://downloads.python.org/pypy/pypy${pythonVersion}-v${version}-aarch64.tar.bz2";
+    x86_64-linux = "https://downloads.python.org/pypy/pypy${pythonVersion}-v${version}-linux64.tar.bz2";
+  };
+
 in with passthru; stdenv.mkDerivation {
   inherit pname version;
 
   src = fetchurl {
-    url = "https://downloads.python.org/pypy/pypy${pythonVersion}-v${version}-linux64.tar.bz2";
+    url = downloadUrls.${stdenv.system} or (throw "Unsupported system: ${stdenv.system}");
     inherit sha256;
   };
 
-  buildInputs = [ which ];
+  buildInputs = deps;
```

```suggestion
  buildInputs = [
    bzip2
    gdbm
    sqlite
    zlib
    expat
    ncurses6
    tcl-8_5
    tk-8_5
  ];
```
now that we don't use deps anywhere anymore